docs/wipy/tutorial/blynk.rst

```diff
@@ -1,19 +1,19 @@
 Getting started with Blynk and the WiPy
 ---------------------------------------
 
-Blynk is a platform with iOS and Android apps to control
-Arduino, Raspberry Pi and the likes over the Internet.
-You can easily build graphic interfaces for all your
-projects by simply dragging and dropping widgets.
+Blynk provides iOS and Android apps to control any hardware over the Internet
+or directly using Bluetooth. You can easily build graphic interfaces for all your projects
+by simply dragging and dropping widgets, right on your smartphone.
+Blynk is the most popular IoT platform used by design studios, makers, educators,
```

Comment:
I think this last sentence should be removed, because it's subjective and more of an endorsement rather than providing technical information.